EIPS/eip-3664.md

```diff
@@ -0,0 +1,308 @@
+---
+eip: 3664
+title: Next generation game NFT
+author: TylerZ (@tylerztl), TY (@nimocat)
+discussions-to: https://ethereum-magicians.org/t/eip-3664-next-generation-game-nft-standard/6738
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-17
+requires: 20, 1155
+---
+
+## Simple Summary
+This is a standard for next generation game NFTs, which extends ERC-1155.
```

Comment:
```suggestion
This is a standard for next generation game NFTs, which extends [ERC-1155](./eip-1155.md).
```
Please update your references to EIPs and ERCs to link to them.

Reply:
Thanks for your suggestions, we will working on details to satisfy the requirements, many thanks!